snapcraft.yaml

```diff
@@ -0,0 +1,26 @@
+name: ponzu
+version: '1.0'
```

Comment:
@igorljubuncic - would it be safer to use the same version number as the one most recently tagged (in this case, `0.10.1`)? Sorry, I'm not sure what a 1.0 version might imply to a snap user.

Reply:
The version string has no semantic meaning in snaps - you can use anything you like. You can even use the git tag if you like, or any naming scheme you want, whatever makes most sense to you.

https://docs.snapcraft.io/snapcraft-yaml-reference